.env.local

```diff
@@ -15,3 +15,5 @@ KAFKA_BROKER_ADDR=localhost:9092
 SC_TEAM_EMAIL='appeals.ch.fake+SC@gmail.com'
 NI_TEAM_EMAIL='appeals.ch.fake+NI@gmail.com'
 DEFAULT_TEAM_EMAIL='appeals.ch.fake+DEFAULT@gmail.com'
+
+CDN_HOST=chs-dev.internal:8080
```

Comment:
do we need to add this here if it's only used when the PIWIK_URL and PIWIK_SITE_ID are set?

Reply:
Good point, not strictly necessary but I know we will have CDN in use very soon.